app/modules/feed-items/components/FeedItem.js

```diff
@@ -4,38 +4,50 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import type { State } from 'types/state';
 import type { Identifier } from 'types/model';
+import _ from 'lodash';
 import { translate } from 'react-i18next';
 import type { TranslatorProps } from 'react-i18next';
 import moment from 'moment';
+import { getUserNameEmailById } from 'modules/users/selectors';
+import { getTitleById } from 'modules/topics/selectors';
+import type { UserNameEmail } from 'modules/users/model';
+import md5 from 'blueimp-md5';
 
 import { Feed } from 'semantic-ui-react';
 
-import professor2 from 'assets/images/avatar/professor2.jpg';
-
 import type { FeedItemType } from '../model';
+
 import { predicateTypes } from '../model';
 import { getById } from '../selectors';
 
+
 type PassedProps = {
   feedItemId: Identifier,
 };
 
 type StateProps = {
   feedItem: FeedItemType,
+  userNameEmail: UserNameEmail,
```

Comment:
Is this a username or an email? The following properties are available from a user:
- `email`
- `firstName`
- `lastName`
- `displayName`: `firstName` and `lastName` if there is one.

I would suggest changing this to either two props or `displayNameAndEmail`.